docs/guide/explanation/the-deployment-process.mdx

```diff
@@ -0,0 +1,53 @@
+# The deployment process
+Deployment begins via one of Zephyr Cloud's bundler plugins.
+When added to your bundler's plugin list, the plugin deploys your bundled applicatioon to a remote Zephyr Cloud instance.
+
+## Deployment lifecycle
+The deployment lifecycle can be split into three stages:
+
+*   [Initial bundling](#initial-bundling)
+*   [Fact gathering](#fact-gathering)
+*   [Snapshot upload and tagging](#snapshot-upload-and-tagging)
+
+### Initial bundling
+The application is bundled according to your bundler's configuration.
+The Zephyr Cloud bundler plugin runs as the last step in the bundling process, initiating the next stage of the process.
```

Comment:
This wording is a bit awkward, rather than saying "initiating the next stage" just name the stage.